requirements.txt

```diff
@@ -1,2 +1,3 @@
-fastapi==0.78.0
-nebula3-python==3.1.0
\ No newline at end of file
+fastapi==0.112.2
+nebula3-python==3.1.0
```

Comment:
we should go with a modern version of nebula3-python, and with modern package management rather than the requirements.txt, but let's merge this first!

Reply:
We are using PDM, too!